charts/csm-authorization/Chart.yaml

```diff
@@ -8,6 +8,9 @@ description: |
   storage enablers for Dell EMC storage products.  CSM for Authorization provides storage and Kubernetes administrators the ability
   to apply RBAC for Dell CSI Drivers.
 dependencies:
+  - name: redis
```

Comment:
Does this reference the local redis-csm chart? It's chart name is redis-csm.

Reply:
For the V1 chart, it's just redis